sorting/merge_sort.c

```diff
@@ -1,139 +1,89 @@
 /**
- * @file
- * @brief Implementation of [merge
- * sort](https://en.wikipedia.org/wiki/Merge_sort) algorithm
+ *@author [Bama Charan Chhandogi](https://github.com/BamaCharanChhandogi)
  */
 #include <stdio.h>
-#include <stdlib.h>
 
-/**
- * @addtogroup sorting Sorting algorithms
- * @{
- */
-/** Swap two integer variables
- * @param [in,out] a pointer to first variable
- * @param [in,out] b pointer to second variable
- */
-void swap(int *a, int *b)
+// Function to merge two sorted subarrays into one sorted array
```

Comment:
why have you decreased the documentation please restore the old documentation

Reply:
I have removed the documentation of the "swap" function. because here I don't use the Swap function. so why I write documentation if it!

Reply:
Understood! Sorry for the confusion.

Reply:
Understood! Sorry for the confusion.